tests/scenario/conftest.py

```diff
@@ -20,9 +23,22 @@ def loki_charm():
     ):
         with patch("socket.getfqdn", new=lambda *args: "fqdn"):
             with patch("lightkube.core.client.GenericSyncClient"):
-                yield LokiOperatorCharm
+                with charm_tracing_disabled():
+                    with patch("subprocess.run"):
```

Comment:
this is a heavy patch. Why did we need it?

Reply:
needed it when we were calling the whole `_configure()` function. After refactoring, we don't need it anymore